plugins/plugin-site/src/styles/base.css

```diff
@@ -992,3 +995,65 @@ a.card:hover {
     max-width: 700px;
   }
 }
+@media (prefers-color-scheme: dark) {
```

Comment:
how does this work for non dark? all the variables are already defined? or just empty?

Reply:
All of them are defined, either in jenkins.io stylesheet or Bootstrap. The bootstrap values are taken from Bootstrap's dark mode (with link color modified).

Reply:
All of them are defined, either in jenkins.io stylesheet or Bootstrap. The bootstrap values are taken from Bootstrap's dark mode (with link color modified).